pkgs/games/ddnet/default.nix

```diff
@@ -92,12 +92,12 @@ stdenv.mkDerivation rec {
     Security
   ]);
 
-  patches = [
-    (fetchpatch {
-      url = "https://github.com/ddnet/ddnet/pull/8517/commits/c840bf45016a30e629f7684df5fab5d07b2c70d5.patch";
-      hash = "sha256-UG7pi0Xh/nAHFEF1RIyNZLewF+NFilTLARbV5oUlftc=";
-    })
-  ];
+#  patches = [
+#    (fetchpatch {
+#      url = "https://github.com/ddnet/ddnet/pull/8517/commits/c840bf45016a30e629f7684df5fab5d07b2c70d5.patch";
+#      hash = "sha256-UG7pi0Xh/nAHFEF1RIyNZLewF+NFilTLARbV5oUlftc=";
+#    })
+#  ];
```

Comment:
Could you remove these line completely?
```suggestion
```

Reply:
very good idea, i removed the commted out lines